src/gcp/apphosting.ts

```diff
@@ -46,33 +47,139 @@ export interface Build {
   state: State;
   error: Status;
   image: string;
+  config?: BuildConfig;
   source: BuildSource;
-  buildLogsUri: string;
-  createTime: Date;
-  updateTime: Date;
   sourceRef: string;
+  buildLogsUri?: string;
+  displayName?: string;
+  labels?: Record<string, string>;
+  annotations?: Record<string, string>;
+  uuid: string;
+  etag: string;
+  reconciling: boolean;
+  createTime: string;
+  updateTime: string;
+  deleteTime: string;
 }
 
-export type BuildOutputOnlyFields = "createTime" | "updateTime" | "sourceRef";
+export type BuildOutputOnlyFields =
+  | "state"
+  | "error"
+  | "image"
+  | "sourceRef"
+  | "buildLogUri"
+  | "reconciling"
+  | "uuid"
+  | "etag"
+  | "createTime"
+  | "updateTime"
+  | "deleteTime";
+
+export interface BuildConfig {
+  minInstances?: number;
+  memory?: string;
+}
 
 interface BuildSource {
-  codeBaseSource?: CodebaseSource;
+  codebase: CodebaseSource;
 }
 
+interface CodebaseSource {
```

Comment:
Pretty brutal we can't reuse protos here :/

Reply:
I would've love to use AI to generate the types 😢